cumulusci.yml

```diff
@@ -77,11 +77,17 @@ tasks:
             path: unpackaged/config/trial
             namespace_tokenize: $project_config.project__package__namespace
 
-    add_picklist_values:
+    add_application_type_picklist_values:
         description: Adds picklist values to the given field, if they don't already exist.
         class_path: tasks.add_picklist_values.AddPicklistValues
         group: 'K-12: Metadata'
-        
+        options:
+            sobject: hed__Application__c
+            field: hed__Application_Type__c
+            values: "Elementary,Middle School,High School"
+            recordTypes: picklist
```

Comment:
What is this? "picklist" doesn't make sense to me here as a record type...

Reply:
so I added this because it was in the specs but it doesn't make sense and isn't a required field @robertbwatson so Im going to remove it.